Mixpanel/MPABTestDesignerConnection.m

```diff
@@ -17,13 +17,14 @@
 #import "MPSwizzler.h"
 
 NSString * const kSessionVariantKey = @"session_variant";
+static NSString * const MPConnBarLoadingAnim = @"MPConnectivityBarLoadingAnim";
```

Comment:
Can we drop the abbreviations and keep it consistent with naming? I hate to be pedantic but these are too different.

Something like `kStartLoadingAnimationKey`?